src/components/page-layout/components/sidebar.js

```diff
@@ -55,45 +54,19 @@ export default class Sidebar extends React.Component {
       frontMatter,
       children,
       parentPath,
-      headings,
       topics,
       layoutConfig
     } = this.props;
 
-    const { accordion } = navigation;
-
+    /* Should we support some of these other features as well?
```

Comment:
@katydecorah I wasn't sure if we should consider passing some of these configuration options in to the new sidebar.

Reply:
I don't think so. The new sidebar should be pretty opinionated to help keep us consistent. It's likely we'll open up more configuration options to the PageLayout aside, though.